### ISSUE TITLE
Support for LSI firmware HBA IT mode controller cards

### ISSUE THREAD

**Issue by restot** [IceWhaleTech/ZimaBoard-BIOS#7]
Referencing for this comment from @GJChester 
https://discord.com/channels/884667213326463016/943513030497558598/1134207617284178010

Would love to see support in zima's bios HBA SAS controllers cards.
IBM H1110 SAS-2 HBA LSI 9211-4i P20 IT Mode for instance seems like a great fit for zimaboard with potential to operate of up to 125 HBA lines, (hard drives). 

Please consider implement that support in future updates

**Comment by restot** [comment]
Tried with IBM H1110 LSA firmware, no luck so far

**Comment by Pyromane** [comment]
Unfortunately, a "Broadcom LSI 9200-8e SAS Controller HBA" does not work either.

**Comment by restot** [comment]
Supermicro AOC-S2308L-L8i not working as well

**Comment by Pyromane** [comment]
@JohnGuan Is there any feedback about this?

**Comment by iferlive** [comment]
Sadly IBM H1110 in IT mode also does not work, it gets power but is not detected in BIOS. 

@JohnGuan, have there been any progress on a BIOS update? Any rollout schedule you can share? 

Sadly there have been no BIOS updates at all, the Zimaboard project started in 2021, is already 2024 and we are still running BIOS 1.0 from launch date, no additional features or issues have been addressed. 

We'd love lo see HBA LSI support since for some of us this one was the sole propuse of purchasing the Zimaboard in the first place.

Thanks in advance
-FR

**Comment by Pyromane** [comment]
> I doubt AMI who make the bios are going to revisit the card so long after it’s hit the end of its support life.

The changes must be made to the ZimaBoard bios, hence the issue here.

**Comment by jarneson33** [comment]
This issue should be closed as the BIOS already supports these cards. Just need to make some changes in the BIOS as seen in this video.
https://youtu.be/ar9Xhd2Wqjk?si=pYGQ1iICnxHF0siG

I have a couple ZimaBlades which use the same BIOS as the ZimaBoard and I have the IBM H1110 in IT mode and can confirm making the changes from the video work.

The video suggests a few different changes. I actually found that just changing PCIe speed to Gen2 and enabling Transmitter Half Swing were the only two settings I needed to change to get the IBM H1110 in IT mode recognized. The other suggested changes might be good to do as well but for minimal changes I found just those two get it recongnized.
